main.js

```diff
@@ -340,6 +340,26 @@ function initializeGamePage() {
   recordSpeedInput.style.marginBottom = '20px';
   recordSpeedDiv.appendChild(recordSpeedInput);
 
+  // Add event listener to record button
+  recordButton.addEventListener('click', () => {
```

Comment:
nice change
please add it to the exiting event listener below in the code

also, could prevent the overlap by using an else if branch after your if

Reply:
I wonder if we had a merge conflict and that's why it's separate now